app/controller/settings.js

```diff
@@ -0,0 +1,12 @@
+var exports = module.exports = {};
+	
+
+exports.config = function (configData, req) {
```

Comment:
I know this is basic stuff, but sometimes I couldn't understand how you were imagining this controller to be.